docs/user-guide/volumes.md

```diff
@@ -456,18 +456,31 @@ More details can be found [here](https://github.com/kubernetes/kubernetes/tree/{
 
 ### vsphereVolume
 
+__Prerequisite: Kubernetes with vSphere Cloud Provider configured. 
+For cloudprovider configuration please refer [vSphere getting started guide](http://kubernetes.io/docs/getting-started-guides/vsphere/).__
+
 A `vsphereVolume` is used to mount a vSphere VMDK Volume into your Pod.  The contents
-of a volume are preserved when it is unmounted.
+of a volume are preserved when it is unmounted. It supports both VMFS and VSAN datastore.
 
-__Important: You must create a VMDK volume using `vmware-vdiskmanager -c` or
-the VSphere API before you can use it__
+__Important: You must create a VMDK volume using either of following ways or
```

Comment:
Combine L465, 466, 470, 471: - You must create VMDK using one of the following method before using with POD;